Readme.md

```diff
@@ -167,3 +167,32 @@ ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 -----------------------------------------------------------------------------
 ```
+
+## Reviewers guide
+
+When reviewing pull request there are common things a reviewer should keep
+in mind.
+
+Reviewing changes to `src/*` and `nuklear.h`:
+
+* Ensure C89 compatibility.
+* The code should work for several backends to an acceptable degree.
+* Recommend simple optimizations.
+  * Pass small structs by value instead of by pointer.
+  * Use local buffers over heap allocation when possible.
+* Check that the coding style is consistent with code around it.
+  * Variable/function name casing.
+  * Indentation.
+  * Curly bracket (`{}`) placement.
+* Ensure that the contributer have bumped the appropriate version in
+  [package.json](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/package.json)
+  and added their changes to the
+  [CHANGELOG](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/src/CHANGELOG).
+* Have at least one other person review the changes before merging.
+
+Reviewing changes to `demo/*`, `example/*` and other files in the repo:
+
+* Less strictness is required here.
+  * The above are nice, but not required if fixing them are none trivial.
```

Comment:
Shouldn't we also list some of the specific suggestions where the review shouldn't be that strict? And at least the overall idea (which is to get backends merged as soon as possible to allow others to use Nuklear at least partially instead of fleeing to other GUI libraries :wink:)? Maybe something along these lines:

* use of modern technology not discouraged (e.g. C11 instead of C89), but PRs changing e.g. C11 to C89 will be merged as they add value to the code
* merge even incomplete or slow or a bit messy stuff, but always comment on these deficiencies in the code directly (or make todo lists at the top of source files)
* the goal is to minimize the time to merge at the cost of slight impact on quality